xonsh/fs.py

```diff
@@ -0,0 +1,94 @@
+"""
```

Comment:
Yeah, I was just copying what `os.py` does. The closure _is_excessive. Although I will still stand by my argument that no part of that is non-trivial. (ie, I think it's more of a memory excess than a CPU excess.)